docs/pages/components/avatar.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Avatar
+id: avatar
+keywords: icon, avatar
```

Comment:
can you also add identifier as a keyword so if ppl search for it they will go to Avatar?